### PR TITLE
[zh-cn] sync security-context

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/tasks/configure-pod-container/security-context.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 1]
```diff
@@ -710,7 +710,27 @@ To assign SELinux labels, the SELinux security module must be loaded on the host
...
 
-{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+{{< feature-state feature_gate_name="SELinuxMountReadWriteOncePod" >}}
+
+{{< note >}}
+<!--
+Kubernetes v1.27 introduced an early limited form of this behavior that was only applicable
+to volumes (and PersistentVolumeClaims) using the `ReadWriteOncePod` access mode.
+-->
+Kubernetes v1.27 引入了此行为的早期限制形式，仅适用于使用 `ReadWriteOncePod`
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:720]:
```suggestion
Kubernetes v1.27 引入了此行为的早期受限形式，仅适用于使用 `ReadWriteOncePod`
```

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 2]
```diff
@@ -734,9 +754,14 @@ To benefit from this speedup, all these conditions must be met:
   [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)。
 
 <!--
-* Pod must use PersistentVolumeClaim with `accessModes: ["ReadWriteOncePod"]`.
+* Pod must use PersistentVolumeClaim with applicable `accessModes` and [feature gates](/docs/reference/command-line-tools-reference/feature-gates/):
+  * Either the volume has `accessModes: ["ReadWriteOncePod"]`, and feature gate `SELinuxMountReadWriteOncePod` is enabled.
+  * Or the volume can use any other access modes and both feature gates `SELinuxMountReadWriteOncePod` and `SELinuxMount` must be enabled.
 -->
-* Pod 必须以 `accessModes: ["ReadWriteOncePod"]` 模式使用 PersistentVolumeClaim。
+* Pod 必须使用带有适用的 `accessModes` 和 [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:761]:
```suggestion
* Pod 必须使用带有对应的 `accessModes` 和[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
```

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 3]
```diff
@@ -774,19 +799,86 @@ The more files and directories in the volume, the longer that relabelling takes.
...
+{{< feature-state feature_gate_name="ProcMountType" >}}
+
+<!--
+For runtimes that follow the OCI runtime specification, containers default to running in a mode where
+there are multiple paths that are both masked and read-only.
+The result of this is the container has these paths present inside the container's mount namespace, and they can function similarly to if
+the container was an isolated host, but the container process cannot write to
+them. The list of masked and read-only paths are as follows:
+-->
+对于遵循 OCI 运行时规范的运行时，容器默认以存在多个屏蔽且只读的路径的模式运行。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:816]:
```suggestion
对于遵循 OCI 运行时规范的运行时，容器默认运行模式下，存在多个被屏蔽且只读的路径。
```

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 4]
```diff
@@ -774,19 +799,86 @@ The more files and directories in the volume, the longer that relabelling takes.
...
+这样做的结果是容器在容器的挂载命名空间内存在这些路径，并且它们的特性与容器是隔离主机时类似，
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:817]:
```suggestion
这样做的结果是在容器的 mount 命名空间内会存在这些路径，并且这些路径的工作方式与容器是隔离主机时类似，
```

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 5]
```diff
@@ -774,19 +799,86 @@ The more files and directories in the volume, the longer that relabelling takes.
...
+但容器进程无法写入它们。
+屏蔽和只读路径列表如下：
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:819]:
```suggestion
被屏蔽的和只读的路径列表如下：
```

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 6]
```diff
@@ -774,19 +799,86 @@ The more files and directories in the volume, the longer that relabelling takes.
...
+
+<!--
+- Masked Paths:
+-->
+- 屏蔽路径：
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:824]:
```suggestion
- 被屏蔽的路径：
```

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 7]
```diff
@@ -774,19 +799,86 @@ The more files and directories in the volume, the longer that relabelling takes.
...
+  - `/proc/timer_list`
+  - `/proc/timer_stats`
+  - `/proc/sched_debug`
+  - `/proc/scsi`
+  - `/sys/firmware`
+
+<!--
+- Read-Only Paths:
+-->
+- 只读路径：
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:840]:
```suggestion
- 只读的路径：
```

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 8]
```diff
@@ -774,19 +799,86 @@ The more files and directories in the volume, the longer that relabelling takes.
...
+对于某些 Pod，你可能希望绕过默认的路径屏蔽。
+最常见的情况是你尝试在 Kubernetes 容器内（在 Pod 内）运行容器。
+
+<!--
+The `securityContext` field `procMount` allows a user to request a container's `/proc`
+be `Unmasked`, or be mounted as read-write by the container process. This also
+applies to `/sys/firmware` which is not in `/proc`.
+-->
+`securityContext` 字段 `procMount` 允许用户请求容器的 `/proc` 为 `Unmasked`，
+或者由容器进程以读写方式挂载。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:862]:
```suggestion
或者由容器进程以读写方式挂载。这一设置也适用于不在 `/proc` 内的 `/sys/firmware` 路径。
```

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 9]
```diff
@@ -774,19 +799,86 @@ The more files and directories in the volume, the longer that relabelling takes.
...
-这个缺陷已经在 Kubernetes 1.26 中修复。
+<!--
+Setting `procMount` to Unmasked requires the `spec.hostUsers` value in the pod
+spec to be `false`. In other words: a container that wishes to have an Unmasked
+`/proc` or unmasked `/sys` must also be in a
+[user namespace](/docs/concepts/workloads/pods/user-namespaces/).
+Kubernetes v1.12 to v1.29 did not enforce that requirement.
+-->
+将 `procMount` 设置为 Unmasked 需要将 Pod 规约中的 `spec.hostUsers`
+的值设置为 `false`。换句话说：希望拥有未屏蔽的 `/proc` 或 `/sys`
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:879]:
```suggestion
的值设置为 `false`。换句话说：希望使用未被屏蔽的 `/proc` 或 `/sys`
```

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 10]
```diff
@@ -774,19 +799,86 @@ The more files and directories in the volume, the longer that relabelling takes.
...
+路径的容器也必须位于[用户命名空间](/zh-cn/docs/concepts/workloads/pods/user-namespaces/)中。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:880]:
```suggestion
路径的容器也必须位于 [user 命名空间](/zh-cn/docs/concepts/workloads/pods/user-namespaces/)中。
```

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 11]
```diff
@@ -872,3 +968,5 @@ kubectl delete pod security-context-demo-4
 * [AllowPrivilegeEscalation 的设计文档（英文）](https://github.com/kubernetes/design-proposals-archive/blob/main/auth/no-new-privs.md)
 * 关于在 Linux 系统中的安全机制的更多信息，可参阅
   [Linux 内核安全性能力概述](https://www.linux.com/learn/overview-linux-kernel-security-features)（注意：部分信息已过时）。
+* 了解 Linux pod 的 [用户命名空间](/zh-cn/docs/concepts/workloads/pods/user-namespaces/)。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:971]:
```suggestion
* 了解 Linux pod 的 [user 命名空间](/zh-cn/docs/concepts/workloads/pods/user-namespaces/)。
```

File: content/zh-cn/docs/tasks/configure-pod-container/security-context.md [hunk 12]
```diff
@@ -872,3 +968,5 @@ kubectl delete pod security-context-demo-4
...
+* [OCI 运行时规范中的屏蔽路径](https://github.com/opencontainers/runtime-spec/blob/f66aad47309/config-linux.md#masked-paths)
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/security-context.md:972]:
```suggestion
* [OCI 运行时规范中的被屏蔽的路径](https://github.com/opencontainers/runtime-spec/blob/f66aad47309/config-linux.md#masked-paths)
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**